### PR TITLE
Fix(Events): Unload once events

### PR DESCRIPTION
**Pull request by PyroTechniac**
Technically gives a small performance boost as well.

### CODE REVIEW

**Review by favna — APPROVED** [review]
Every little bit matters 👌

File: src/events/readySweeper.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ const HEADER = `\u001B[39m\u001B[94m[MEMORY CLEANUP]\u001B[39m\u001B[90m`;
 export default class extends Event {
 
 	public constructor(store: EventStore, file: string[], directory: string) {
-		super(store, file, directory, { event: 'ready' });
```
Comment (cfanoulis) [inline src/events/readySweeper.ts:9]: I... don't think that runs only once

**Review by cfanoulis — CHANGES_REQUESTED** [review]
Hmmm... something's not quite right

File: src/events/readySweeper.ts (hunk 1 above)
Comment (PyroTechniac) [reply src/events/readySweeper.ts:9]: It's for the `ready` event, which is only fired once

Comment (favna) [reply src/events/readySweeper.ts:9]: Ready is only triggered once, specifically, when the bot is ready. It is triggered immediately before KlasaReady.

Comment (cfanoulis) [reply src/events/readySweeper.ts:9]: whoops, didn't see that

/resolved

**Review by cfanoulis — APPROVED** [review]
surprised this isn't a thing already :omegaLOL:

**Review by kyranet — COMMENTED** [review]
Nice catch, thanks you!

**Review by kyranet — APPROVED** [review]

**Pull request merged**